src/AspNet.Security.OpenId/Events/OpenIdRedirectContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OpenId.Events
+{
+    public class OpenIdRedirectContext : PropertiesContext<OpenIdAuthenticationOptions>
+    {
+        public OpenIdRedirectContext(HttpContext context,
+            AuthenticationScheme scheme,
+            OpenIdAuthenticationOptions options,
+            AuthenticationProperties properties, OpenIdAuthenticationMessage message)
+            : base(context, scheme, options, properties)
+        {
+            ProtocolMessage = message;
+        }
+
+        /// <summary>
+        /// Gets the protocol message that has been generated by the handler up to this point.
+        /// </summary>
+        public OpenIdAuthenticationMessage ProtocolMessage { get; set; }
+
+        /// <summary>
+        /// If true, will skip any default logic for this redirect.
+        /// </summary>
+        public bool Handled { get; private set; }
```

Comment:
Where is this property used? I don't see any logic in the handler that relies on it.

Reply:
Removed - I used the OpenIdConnect implementation as a template. Not sure exactly how it's being consumed but appears to be called from tests.